qubesbuilder/plugins/source_rpm/scripts/prepare-chroot-base

```diff
@@ -181,11 +173,11 @@ if ! [ -f "${INSTALL_DIR}/tmp/.prepared_base" ]; then
     echo "INFO: Verifying signatures..."
     set +x
     for file in "${DOWNLOAD_DIR}"/*; do
-        result=$(rpmkeys "${RPM_OPTS[@]}" --root="${INSTALL_DIR}" --checksig -- "${file}") || {
```

Comment:
Is this the host or the build RPM?  I recommend using `rpmcanon` especially in the latter case